app/models/graph/types/planet.rb

```diff
@@ -0,0 +1,20 @@
+module Graph
+  module Types
+    Planet = GraphQL::ObjectType.define do
+      name "Planet"
+      description "A planet in the StarWars universe"
+
+      field :id, types.ID
+
+      field :name, types.String
+      field :diameter, types.Int
+      field :rotationPeriod, types.Int, property: :rotation_period
+      field :orbitalPeriod, types.Int, property: :orbital_period
+      field :gravity, types.String
+      field :population, types.Int
```

Comment:
We'll want this to be a float as it represents "millions of citizens". Some planets have less than a million.